### DISCUSSION TITLE
how to fix this maven complain, I'm a green.Thanks

### DISCUSSION THREAD

**Discussion by for-pyja** [dbeaver/dbeaver#18404] [DBeaver Development]
### Description

maven complaining that "P2ResolverFactory service is missing"
![image](https://user-images.githubusercontent.com/61376061/205117673-dd12138b-7024-4b76-9dc5-df31ac2a13b1.png)

### DBeaver Version

latest

### Operating System

windows10  java17.0.5 maven3.8.6

**Comment by ShadelessFox** [comment]
Hello @for-pyja,

You're not supposed to build/sync Maven projects in Intellij IDEA. Instead, everything should be done in Eclipse.

After that, you can open the project in IDEA and just build it as a regular project.
